rewriter/Minitest.cc

```diff
@@ -420,28 +533,97 @@ ast::ExpressionPtr runSingle(core::MutableContext ctx, bool isClass, ast::Send *
         return constantMover.addConstantsToExpression(send->loc, move(method));
     }
 
+    if (send->numPosArgs() == 0 && send->fun == core::Names::subject()) {
+        auto declLoc = send->funLoc;
+        return ast::MK::SyntheticMethod0(send->loc, declLoc, core::Names::subject(), std::move(block->body));
+    }
+
+    // Handle include_context with block (1+ pos args + optional kwargs/block)
+    if (send->fun == core::Names::includeContext() && send->numPosArgs() >= 1) {
```

Comment:
```suggestion
    if (send->fun == core::Names::includeContext() && send->numPosArgs() == 1) {
```

Should this check for exactly one arg?

You do not have any tests that say that two positional args to includeContext are expected.